openwisp_monitoring/check/settings.py

```diff
@@ -5,9 +5,13 @@
     (
         ('openwisp_monitoring.check.classes.Ping', 'Ping'),
         ('openwisp_monitoring.check.classes.ConfigApplied', 'Configuration Applied'),
+        ('openwisp_monitoring.check.classes.Iperf', 'Iperf'),
     ),
 )
 AUTO_PING = get_settings_value('AUTO_PING', True)
 AUTO_CONFIG_CHECK = get_settings_value('AUTO_DEVICE_CONFIG_CHECK', True)
 MANAGEMENT_IP_ONLY = get_settings_value('MANAGEMENT_IP_ONLY', True)
 PING_CHECK_CONFIG = get_settings_value('PING_CHECK_CONFIG', {})
+# By default it should be disabled.
+AUTO_IPERF = get_settings_value('AUTO_IPERF', True)
```

Comment:
Is this a reminder? We should flip  it back to False, and set this value in settings.py

Reply:
Yes it should be `false`.